src/test/ui/nll/issue-52663-span-decl-captured-variable.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0507]: cannot move out of captured variable in an `Fn` closure
+  --> $DIR/issue-52663-span-decl-captured-variable.rs:20:26
+   |
+LL |        expect_fn(|| drop(x.0));
```

Comment:
shouldn't we highlight something here?

Reply:
Oops, forgot to update the label logic with the same changes that made that branch get used in the first place for complex cases like this one. Fixed now.